utils/v2dl3-vegas-docker/Dockerfile_vegas_v2dl3_arm64

```diff
@@ -0,0 +1,144 @@
+### Dockerfile for building an ubuntu image with VEGAS build
```

Comment:
As you run the actions on ubuntu-latest, do you really need the dockerfiles for the different architectures?

Reply:
This is just for users or developers to build or reference an environment set up for v2dl3-vegas, since it can be pretty difficult. The CI environment has its own recipe. The x64 and arm64 recipes install v2dl3 for the user as well. The CI recipe doesn't since the github action handles installation.